introduction.rst

```diff
@@ -49,11 +49,7 @@ The different applications in the image above are:
   from Bintray, doesn't require an account, an account is only needed to upload packages. Besides
   that, Bintray provides a central repository called `conan-center
   <https://bintray.com/conan/conan-center>`_ which is moderated, and packages are reviewed before
-  being accepted to ensure quality. This repository is currently empty, the process has just
-  started. In the meantime, the repository `conan-transit
-  <https://bintray.com/conan/conan-transit>`_ contains a copy of all the packages that were in the
```

Comment:
I would leave the reference to conan-transit in the docs, at least until it is removed. Explicitly and clearly say that it is deprecated, most package recipes will be broken and it will be removed and closed soon.

Reply:
This PR is the documentation for a PR removing conan-transit indeed...

Reply:
But one thing is to remove conan-transit as a default remote in the client, and another is to close conan-transit. ``conan-transit`` will be still active for a while, containing recipes and packages. Many of them might be broken, but there is a possibility that someone still using some working. So it is good to leave a reference to conan-transit in the docs, no need that it completely dissappears now.